egui/src/widgets/plot.rs

```diff
@@ -425,9 +470,15 @@ impl Widget for Plot {
             vec2(width, height)
         };
 
-        let (rect, response) = ui.allocate_exact_size(size, Sense::hover());
+        let (rect, response) = ui.allocate_exact_size(size, Sense::drag());
 
-        let mut bounds = bounds;
+        if response.double_clicked_by(PointerButton::Primary) || bounds == Bounds::EMPTY {
```

Comment:
There is a potential bug here: the `bounds` set by the builder is constrained by `include_x` and/or `include_y`. If the user calls `include_y(0.0)` to always show the X-axis, then the bounds won't `== EMPTY`, but they will be invalid. So the check should be `|| !bounds.is_valid()` with something like:

``` rust
impl Bounds {
    fn is_valid(&self) -> bool {
        self.is_finite() && self.width() > 0.0 && self.height() > 0.0
    }
}
```

Reply:
Fixed in 1b00e7f